network/transport/types.go

```diff
@@ -63,6 +63,16 @@ func (p Peer) ToFields() logrus.Fields {
 	}
 }
 
+// Key returns a unique key for this Peer including PeerID and NodeDID.
+// Usable as map index, not usable for presentation.
+func (p Peer) Key() string {
+	// address is included since 2 connections may exist for a peer (inbound/outbound)
+	if p.NodeDID.Empty() {
+		return fmt.Sprintf("%s@%s", p.ID, p.Address)
```

Comment:
What if I make a peer ID:

```
169EAA84-FB3A-4D25-8340-6576787A0D84(did:nuts:some-did)
```

Then I can still mangle with the connection manager?

Safer to not have conditional formatting, but have a set format (just always print nodeDID inbetween `(, )`), even when its empty. Also simpler/less code to test.